### PR TITLE
fix broken links

### PR DESCRIPTION
**Pull request by ashishpatelcs**
**Description of the change**: fixed broken links. these links on github, leads user to a 404 page on github.
**Reason for the change**:
**Link to original source**:

Closes #

### CODE REVIEW

File: content/docs/ui/sending-email/working-with-marketing-templates.md [hunk 1]
```diff
@@ -76,6 +76,6 @@ You can easily duplicate a pre-built template provided by SendGrid.
 
  ## 	Additional Resources
 
-- [Sending an Email]({{root_url}}/ui/sending-email/how-to-send-email-with-marketing-campaigns/)
-- [Design Editor]({{root_url}}/ui/sending-email/editor/#-The-Design-Editor)
-- [Code Editor]({{root_url}}/ui/sending-email/editor/#the-code-editor)
```
Comment (pushkyn) [inline content/docs/ui/sending-email/working-with-marketing-templates.md]: @ashishpatelcs why did you decide that these are broken links? They work great on https://sendgrid.com/docs/ui/sending-email/working-with-marketing-templates/ .
The `root_url` word will be replaced with the correct prefix when building the project.

**Comment by pushkyn** [comment]
@ashishpatelcs why did you decide that these are broken links? They work great on https://sendgrid.com/docs/ui/sending-email/working-with-marketing-templates/ .
The root_url word will be replaced with the correct prefix when building the project.

**Comment by ashishpatelcs** [comment]
@pushkyn Hi, When i clicked the link on Github, it sent me to a 404 github page.

**Comment by ksigler7** [comment]
@ashishpatelcs - oooohhhhh! I was wondering why I kept getting so many PRs like this - thanks for explaining! Yeah, this is designed to work here: https://sendgrid.com/docs/ui/sending-email/working-with-marketing-templates/ and not necessarily in github. We have it relative because we have a local and staging build as well, and we want the links to work in those builds. We don't care that they don't work in github, it's much more important that it works in every stage of our build.

**Comment by ksigler7** [comment]
@ashishpatelcs - oooooh, I was wondering why I had gotten a ton of PRs like these! This is a relative link and it is designed to work in local, staging and production builds of our site and not necessarily in github (here's this page in production: https://sendgrid.com/docs/ui/sending-email/working-with-marketing-templates/). We don't care that they don't work in github, it's much more important that it works in every stage of our build.

**Pull request closed without merging**